components/Dialogs/src/MDCAlertController+ButtonForAction.h

```diff
@@ -0,0 +1,33 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MaterialDialogs.h"
+
+@interface MDCAlertController (ButtonForAction)
+
+/**
+ Returns an MDCButton associated with the given action. This method might create the button if
+ it no associated button exist for the action yet. Buttons returned by thismethod may not (yet)
```

Comment:
tip: typo in comment: remove "it" in sentence.

Reply:
Please file a bug to address this in a follow-up. We can't make changes in the release.

Reply:
also s/thismethod/this method/